app/custom-oauth/server/custom_oauth_server.js

```diff
@@ -217,6 +218,7 @@ export class CustomOAuth {
 		});
 	}
 
+	// eslint-disable-next-line complexity
```

Comment:
If this is just spawning a warning on eslint, it's better to remove it.

Reply:
Okay